contactListApp/server.js

```diff
@@ -14,15 +14,19 @@ mongoose.connect( 'mongodb://localhost/contactList', function (err) {
 //mongoose.connect('mongodb://127.0.0.1:27017/test/contactList');﻿
 
 var contactListSchema = mongoose.Schema({
-name: String, 
+name: String,
 email: String,
 number: String
 }, {collection: 'contactList'});
 
-var ContactList = mongoose.model('ContactList', contactListSchema);﻿
+var contactList = mongoose.model('ContactList', contactListSchema);﻿
 
-
-var ContactList = mongoose.model('contactList', contactListSchema);
+app.get("/contactList", function(req, res) {
+  contactList.find({}, function(err, result) {
+    if(err) res.status(500).json(err);
+    else res.status(200).json(result);
+  });
+});
```

Comment:
This is the router. It will give a 500 error if there is an issue with the server, or a 200 if the request is successful. You can read more about status codes at [https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#5xx_Server_Error](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#5xx_Server_Error). Part of RESTful programming is displaying the appropriate status at the right time, and it is good to memorize the more common ones.